frontend/src/store/auth/index.js

```diff
@@ -27,7 +27,10 @@ const getters = {
   userAuthObject: ({ userAuthObject }) => userAuthObject,
   loading: ({ loading }) => loading,
   error: ({ error }) => error,
-  authCV: ({ authCV }) => authCV
+  authCV: ({ authCV }) => {
```

Comment:
This doesn't work. It'll just alter the param, not the underlying CV.

I suggest this:

```js
const _authCV = new CV()

const state = {...}

const getters = { authCV: () => _authCV }
const mutations = { ...SIGNAL_AUTH_CV() { _authCV.signal() }}
```

Reply:
can't i just not destructure state? that seems to work
```
  authCV: (state) => {
    if (state.authCV === null) state.authCV = new CV()
    return state.authCV
  }
```

Reply:
Also this is why we should be using Rust for our frontend ^_^